### ISSUE TITLE
Block movers don't work so well with speech recognition software

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5750]
Thanks to the in-depth testing made by @ewaccess there's an interesting issue worth considering.

Speech recognition software accept voice commands and to understand what is the element in a web page the command refers to, they use the element accessible name. In simple words, the so called "accessible name" can be a button text, a link text, an UI control `aria-label`, etc. Basically tje text or the attribute that gives a control its "name".

Of course, speech recognition software users need to know the name of a control to be able to voice commands. Ideally, any control should have a visible "name". In many parts of the UI Gutenberg uses "tooltips" to expose a control `aria-label`. That's a trade-off with design needs that works to some extent. 

Regardless, in the specific case of the block movers, the UI is visually displaying this tooltip:

<img width="544" alt="screen shot 2018-03-22 at 17 08 50" src="https://user-images.githubusercontent.com/1682452/37783820-b0ddbb66-2df6-11e8-8c3a-414c821640d2.png">

and of course "Move down" for the other button. 

However, the displayed tooltip text is different from the actual label. Users will try the following voice command because the interfaces instructs them to do so:

`Click Move up`

while the actual accessible name given by the `aria-label` is, for example:

`Move "Paragraph" block from position 2 up to position 1`

So in the specific test run by @ewaccess with Dragon NaturallySpeaking, this won't work and Dragon won't execute the command.

As recommended by the WCAG 2.1, the accessible name must contain the text presented visually. See https://www.w3.org/TR/WCAG21/#label-in-name
>For user interface components with labels that include text or images of text, the name contains the text presented visually.
> A best practice is to have the text of the label at the start of the name.

It is important to note that this applies to all UI controls. 

See also the related "Understanding Label in Name"
https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html

**Comment by afercia** [comment]
Basically, this is a wording issue. There's the need to prepend "Move down" and "Move up" the the aria-labels, and ensure that will be correct also in other languages.
